src/main/constraints/00_71_container_settings_fkey.sql

```diff
@@ -0,0 +1,6 @@
+SET search_path = public, pg_catalog;
+
+ALTER TABLE ONLY container_settings
+    ADD CONSTRAINT container_settings_interactive_apps_proxy_settings_id_fkey
+    FOREIGN KEY(interactive_apps_proxy_settings_id)
```

Comment:
So far we've been putting `FOREIGN KEY` constraints in the regular `src/main/constraints/XX_table_name.sql` files, and only primary keys (since they have to be defined before other constraints) have been going into `constraints/00_XX_table_name.sql` files.

Maybe this should just go into a `constraints/71_container_settings.sql` file for consistency?